docs/protocol.md

```diff
@@ -307,6 +307,8 @@ The meaning of the three args depends on the systems on each end. The format of
 arg1, arg2, and arg3 is unspecified at the transport level. These are opaque
 binary blobs as far as tchannel is concerned.
 
+The size of `arg1` is at most 16 kilobytes.
```

Comment:
Please spell this `16KiB`.

The phrase "16 kilobytes" is open to interpretation, but "16KiB" means `16 * 1024 = 16384`